tests/zfs-tests/tests/functional/l2arc/l2arc_arcstats_pos.ksh

```diff
@@ -70,19 +70,59 @@ log_must fio $FIO_SCRIPTS/mkfiles.fio
 log_must fio $FIO_SCRIPTS/random_reads.fio
 
 log_must zpool offline $TESTPOOL $VDEV_CACHE
+
+typeset do_once=true
+while $do_once || [ $l2size1 -ne $l2size2 ]; do
+	typeset l2size1=$(get_arcstat l2_size)
+	sleep 2
+	typeset l2size2=$(get_arcstat l2_size)
+	do_once=false
+done
```

Comment:
nit: since all of these blocks the same moving this to a helper function might be nice.  Many of the test groups include a shared `.kshlib` file which is sourced to provide some useful functions.

Reply:
Yes, will do!